python/pyspark/sql/window.py

```diff
@@ -217,11 +242,16 @@ def rangeBetween(self, start, end):
         :param end: boundary end, inclusive.
```

Comment:
I think we should also update the description above `We recommend user ...` in a similar way. Linking is optionally better but just ` ``...`` ` is fine enough to me.